source/MRMesh/MRMeshBoolean.h

```diff
@@ -55,4 +55,16 @@ struct BooleanResult
 MRMESH_API BooleanResult boolean( const Mesh& meshA, const Mesh& meshB, BooleanOperation operation,
                                   const AffineXf3f* rigidB2A = nullptr, BooleanResultMapper* mapper = nullptr );
 
+/** \brief Returns points that represents the vertices of mesh intersection
```

Comment:
probably better rename vertex->point, since they are neither vertices of mesh A nor of mesh B, or say that these are vertices of imagined contour